docs/plot_result.rst

```diff
@@ -0,0 +1,34 @@
+####################
+Plotting fit results
+####################
+
+Plotting
+========
+
+The saved results from a PAHFIT run can be plotted using the ``plot_pahfit``
+script.  By default, the plot is displayed.  To save the plot instead,
+use the "--savefig" command line option.
+
+.. code-block:: console
+
+  $ plot_pahfit M101_Nucleus_irs.ipac M101_Nucleus_irs_output.ipac
+
+The first argument gives the file with the observed spectrum.
+The second argument gives the file with the model pack file to be used.
```

Comment:
Isn't the second argument the output result of the fit?

Reply:
Yep.

Reply:
Docs updated.